.github/workflows/git-artifacts.yml

```diff
@@ -0,0 +1,316 @@
+name: git-artifacts
+
+on:
+  # Uncomment the "push:" line if you want to build artifacts without sending
+  # a `repository_dispatch` event, and possibly set the environment variables
+  # `BUILD_ONLY`, `REPOSITORY` and `REF` explicitly.
+  #
+  # push:
+  repository_dispatch:
+    types:
+      - git-artifacts
+
+env:
+  GIT_CONFIG_PARAMETERS: "'user.name=${{github.actor}}' 'user.email=${{github.actor}}@users.noreply.github.com' 'user.signingkey=mine'"
+  GPG_OPTIONS: "--batch --yes --no-tty --list-options no-show-photos --verify-options no-show-photos --pinentry-mode loopback"
+  HOME: "${{github.workspace}}\\home"
+  MSYSTEM: MINGW64
+  PACKAGER: ${{github.actor}} <${{github.actor}}@users.noreply.github.com>
+  USERPROFILE: "${{github.workspace}}\\home"
+  BUILD_ONLY: "${{github.event.client_payload.build_only}}"
+  REPOSITORY: "${{github.event.client_payload.repository}}"
+  REF: "${{github.event.client_payload.ref}}"
+
+jobs:
+  bundle-artifacts:
+    runs-on: windows-latest
+    steps:
+      - name: Download git-sdk-64-build-installers
+        shell: bash
+        run: |
+          a=git-sdk-64-build-installers && mkdir -p $a && curl -# https://wingit.blob.core.windows.net/ci-artifacts/$a.tar.xz | tar -C $a -xJf - &&
```

Comment:
Should `git-sdk-64-build-installers` be a job env?  It is used a few times.

Is the `wingit` blob a copy of git-sdk-64?  Sorry I did not download.  If it is, should there be release of git-sdk-64 that can be used instead?

Reply:
The job level env vars for directories could be used in other jobs too (nuget?).  Just getting going on this review while also learning actions.